src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
+    "transport": "http",
+    "icon":"http://google.tieto.com/images/services/google-cloud-platform/BigQuery_500px-1f2fdb56.png",
+    "clients": [
```

Comment:
How hard would it be to introduce this in the future if we do want multiple clients? Either way, there is information that belongs to the API as a whole, and some that is specific to the client library. It seems useful to distinguish that.